xonsh/base_shell.py

```diff
@@ -157,13 +157,6 @@ def readline(self, size=-1):
         self.std.seek(loc + len(s))
         return s
 
-    def write(self, s):
-        """Write a string to both streams and return the length written to the
-        in-memory stream.
-        """
-        self.std.write(s)
-        return self.mem.write(s)
```

Comment:
OMG @gforsyth, you are totally write(). I don't think that it matters much which one is used.  Let's proceed with this version.  Sorry I was AWOL the past few days.